src/rgw/rgw_crypt.cc

```diff
@@ -24,6 +24,19 @@
 #include <cryptopp/cryptlib.h>
 #include <cryptopp/modes.h>
 #include <cryptopp/aes.h>
+
+/*
+ * patching for https://github.com/openssl/openssl/issues/1437
+ * newer versions of civetweb do not experience problems
+ */
+extern "C"
+{
+bool SSL_CTX_set_ecdh_auto(void* dummy, int onoff)
```

Comment:
thanks, this addresses the issue mentioned at https://github.com/ceph/ceph/pull/13315#issuecomment-283617562, but shall we just update the civetweb submodule?